inetum-realjobs-backend/src/main/resources/data-postgres.sql

```diff
@@ -1,512 +1,553 @@
+INSERT INTO public.country (id, name)
+VALUES (237, 'Vanuatu');
+INSERT INTO public.country (id, name)
+VALUES (238, 'Wallis And Futuna');
+INSERT INTO public.country (id, name)
+VALUES (239, 'Samoa');
+INSERT INTO public.country (id, name)
+VALUES (240, 'Kosovo');
+INSERT INTO public.country (id, name)
+VALUES (241, 'Yemen');
+INSERT INTO public.country (id, name)
+VALUES (242, 'Mayotte');
+INSERT INTO public.country (id, name)
+VALUES (243, 'South Africa');
+INSERT INTO public.country (id, name)
+VALUES (244, 'Zambia');
+INSERT INTO public.country (id, name)
+VALUES (245, 'Zimbabwe');
+ALTER SEQUENCE country_id_seq RESTART WITH 246;
+
 
 INSERT INTO public.company (id, changed_by, changed_on, created_by, created_on, version, city, industry, logo, name,
                             country_id)
 VALUES (1, 'initial', '2021-12-27 17:04:17.000000', 'initial', '2021-12-27 17:04:19.000000', 1, 'Huizingen', 'IT', null,
         'Realdolmen', 20);
 ALTER SEQUENCE company_id_seq RESTART WITH 2;
 
+INSERT INTO public.account (id, changed_by, changed_on, created_by, created_on, version, box, city, house_number,
+                            postal_code, street_name, birth_date, email, first_name, gender, last_name, mobile_phone,
+                            password, profile_picture, country_id)
+VALUES (1, 'initial', '2021-12-28 14:21:12.000000', 'initial', '2021-12-28 14:21:15.000000', 0, null, 'Brussels', '7',
+        '1000', 'Road', '2021-12-01', 'user@user.user', 'User', 'MALE', 'User', null,
+        '$2a$10$wrvu9zLKCX7zFtaLBFNVKeBh8Y..TLZCuvnzkUcL7N7/OjTD.ximq', '-_-', 1);
+INSERT INTO public.account (id, changed_by, changed_on, created_by, created_on, version, box, city, house_number,
+                            postal_code, street_name, birth_date, email, first_name, gender, last_name, mobile_phone,
+                            password, profile_picture, country_id)
+VALUES (3, 'initial', '2021-12-28 14:21:14.000000', 'initial', '2021-12-28 14:21:16.000000', 0, null, 'Brussels', '7',
+        '1000', 'Road', '2021-12-01', 'recruiter@realdolmen.com', 'User', 'MALE', 'User', null,
+        '$2a$10$wrvu9zLKCX7zFtaLBFNVKeBh8Y..TLZCuvnzkUcL7N7/OjTD.ximq', null, 1);
+INSERT INTO public.account (id, changed_by, changed_on, created_by, created_on, version, box, city, house_number,
+                            postal_code, street_name, birth_date, email, first_name, gender, last_name, mobile_phone,
+                            password, profile_picture, country_id)
+VALUES (2, 'initial', '2021-12-28 14:21:14.000000', 'initial', '2021-12-28 14:21:16.000000', 0, null, 'Brussels', '7',
+        '1000', 'Road', '2021-12-01', 'recruiter@freelance.com', 'User', 'MALE', 'User', null,
+        '$2a$10$wrvu9zLKCX7zFtaLBFNVKeBh8Y..TLZCuvnzkUcL7N7/OjTD.ximq', null, 1);
+ALTER SEQUENCE account_id_seq RESTART WITH 4;
+
+INSERT INTO public.job_seeker (id, resume_id)
+VALUES (1, null);
+INSERT INTO public.recruiter (id, company_id)
+VALUES (2, null);
```

Comment:
Resume will be made mandatory in #49, please add resumes here (they can be empty)